workflows/sanger-variant-calling/jtracker/workflow/main.jt

```diff
@@ -0,0 +1,508 @@
+# This workflow
+
+workflow:
+  name:  sanger-variant-calling
+  version: "0.1.0"
+
+  input:
+    song_collab_url:
+      type: string
+      default: "https://song.cancercollaboratory.org"
+    song_aws_url:
+      type: string
+      default: "https://virginia.song.icgc.org"
+    reference:
+      type: string
+      is_file: true
+      default: "[reference]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/core_ref_GRCh38_hla_decoy_ebv.tar.gz"
+    annot:
+      type: string
+      is_file: true
+      default: "[annot]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/VAGrENT_ref_GRCh38_hla_decoy_ebv_ensembl_91.tar.gz"
+    snv_indel:
+      type: string
+      is_file: true
+      default: "[snv_indel]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/SNV_INDEL_ref_GRCh38_hla_decoy_ebv-fragment.tar.gz"
+    cnv_sv:
+      type: string
+      is_file: true
+      default: "[cnv_sv]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/CNV_SV_ref_GRCh38_hla_decoy_ebv_brass6+.tar.gz"
+    qcset:
+      type: string
+      is_file: true
+      default: "[qcset]http://ftp.sanger.ac.uk/pub/cancer/dockstore/human/GRCh38_hla_decoy_ebv/qcGenotype_GRCh38_hla_decoy_ebv.tar.gz"
+    exclude:
+      type: string
+      default: "chrUn%,HLA%,%_alt,%_random,chrM,chrEBV"
+    species:
+      type: string
+      default: "human"
+    assembly:
+      type: string
+      default: "GRCh38"
+    skipqc:
+      type: boolean
+      default: false
+    pindelcpu:
+      type: integer
+    cavereads:
+      type: integer
+      default: 800000
+    purity:
+      type: float
+      default: 1.0
+    ploidy:
+      type: float
+      default: 3.65
+    num_threads:
+      type: integer
+      default: 18
+    ref_file:
+      type: string
+      is_file: true
+      default: "[GRCh38_hla_decoy_ebv.fa.fai]https://object.cancercollaboratory.org:9080/swift/v1/genomics-public-data/reference-genome/GRCh38_hla_decoy_ebv/GRCh38_hla_decoy_ebv.fa.fai"
```

Comment:
The `ref_file` refer to `fai` file instead of `fa` file.
Please see: https://github.com/cancerit/PCAP-core/blob/17e202064f774029266edf326cb096c3747098f2/c/bam_stats.c#L62

Reply:
I don't understand this then, isn't `ref_file` used for `cram` input? How will it help to get the actual sequence if no `.fa` file is provided? It might be other hidden way `.fa` file is inferred by `.fai` file